packages/discovery-provider/src/api/v1/metrics.py

```diff
@@ -158,12 +164,14 @@ def get(self):
         return response
 
 
-@ns.route("/aggregates/routes/trailing/month", doc=False)
+@ns.route("/aggregates/routes/trailing/<string:time_range>", doc=False)
```

Comment:
is it possible to document the acceptable string types? ie month/year?

Reply:
Will check other files - there probs is